Feisty/UIComponents/FloatingActionButtons/FloatingActionButton.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  FloatingActionButton.swift
+//  Feisty
+//
+//  Created by Tiewhan Smith on 2020/04/15.
+//  Copyright © 2020 DVT. All rights reserved.
+//
+
+import UIKit
+
+class FloatingActionButton: UIButton {
+
+  private let nibName = "FloatingActionButton"
+  
+  @IBOutlet weak var button: UIButton!
+  
+  var actionToPerform: (() -> Void)?
+  
+  required init?(coder: NSCoder) {
+    super.init(coder: coder)
+    commonInit()
+  }
+  
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+    commonInit()
+  }
+  
+  private func commonInit() {
+    
+    guard let view = loadViewFromNib() else { return }
+    view.frame = self.bounds
+    self.addSubview(view)
+    
+    button.setUpDropShadow(opacity: 0.5)
+    button.setUpCornerRadiusWith(radiusOf: 25)
+    
+  }
+  
+  private func loadViewFromNib() -> UIView? {
+    
+    let nib = UINib(nibName: nibName, bundle: nil)
```

Comment:
I think you had an extension earlier which would have let you say `self.className` instead of needing to hardcode the string

Reply:
I do, I don't know how you remembered it but I have made the change.